packages/fxa-auth-server/lib/routes/devices-and-sessions.js

```diff
@@ -270,7 +270,11 @@ module.exports = (
           },
         },
         response: {
-          schema: {},
+          schema: {
+            enqueued: isA.boolean().optional(),
+            notified: isA.boolean().optional(),
+            notifyError: isA.string().optional(),
+          },
```

Comment:
Here's the actual fix; I made them all optional since they're only an FYI for the client to include in e.g. detailed log files.